hienhuynhxuan/python_unittest
# Hien: Update new test

1. test case: test_get_photo_description
- add new test test_get_photo_description:
+ test_get_photo_description
+ test_get_photo_related

2. test case:  test_update_a_photo
- add new test test_update_a_photo

3. update commented on this pull request.
- Should check more about api response status code.
- Name of Test Class should be should be meaningful.
- No need to commit this file.
- import multiple items.
